architecture/core_objects/routing.adoc

```diff
@@ -370,31 +380,29 @@ repository to run an HAProxy instance alongside the template router plug-in. To
 test routes, an install command is provided.
 
 
-Check the default router ("router"):
-
-****
-$ `oadm router --dry-run`
-****
+To check the default router (named "router"):
```

Comment:
FWIW, I've been using bold (`*`) markup for objects (services, pods, etc) that have been given a name, like in this case. My thinking being that it would seem to fall under the generic "system item" as seen in:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference